README.md

```diff
@@ -13,21 +13,31 @@ Bash Scripts (*.sh)
 -------------------
 
 These scripts will install PyNE and its dependencies on the system.
-Install scripts are available for different versions of both Ubuntu and
-Mint operating systems. The script used to install PyNE should correspond
-to the user's operating system and version. The intention of these
+Install script is available for different versions of both Ubuntu and
+Mint operating systems. The intention of these
 scripts is that PyNE can be ready to use on a clean install of any of
 the supported operating systems. Furthermore, the user should choose either
```

Comment:
Should we define "clean"?  This is mostly about python environments, right?  Should we say that if users have customized/configured their python environment beyond the original OS installation, we can't guarantee that the scripts will work?